lms/djangoapps/ccx/views.py

```diff
@@ -234,6 +234,16 @@ def override_fields(parent, data, graded, earliest=None, ccx_ids_to_delete=None)
                 graded[block.format] = graded.get(block.format, 0) + 1
 
             children = unit.get('children', None)
+            # For a vertical override start and due dates of its all problems.
+            if unit['category'] == u'vertical':
+                for component in block.get_children():
+                    if component.location.category == 'problem' and getattr(component, 'has_score', False):
```

Comment:
I dont think so, it is key. see https://github.com/edx/edx-platform/blob/master/lms/djangoapps/courseware/courses.py#L459